pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart

```diff
@@ -927,3 +915,42 @@ ${compileResult.stdout}
 extension on Uri {
   Uri get parent => File(toFilePath()).parent.uri;
 }
+
+/// Parses depfile contents.
+///
+/// Format: `path/to/my.dill: path/to/my.dart, path/to/more.dart`
+///
+/// However, the spaces in paths are escaped with backslashes, and the
+/// backslashes are escaped with backslashes:
+///
+/// ```dart
+/// String _escapePath(String path) {
+///   return path.replaceAll('\\', '\\\\').replaceAll(' ', '\\ ');
+/// }
+/// ```
+List<String> parseDepFileInputs(String contents) {
+  final output = contents.split(': ').first;
```

Comment:
Consider using `contents.indexOf(': ')` instead here - we're not actually using the `output` variable for anything else than to find the index.